### ISSUE TITLE
Option to highlight all maps and elevations in a collection by default

### ISSUE THREAD

**Issue by bviktor** [OpenGIS/waymark-wp#50]
So I'm using collections to show sections of a long trail together. It'd be really useful to have an option to highlight the complete track by default, especially because it'd show the elevation map for the whole track series, not just for the last map in the collection. To visualize the issue:

![image](https://github.com/user-attachments/assets/59faebe0-0539-4329-9e1f-10b18e7a95e5)

So the map shows a route that's actually 73 km long with 616 / 1343 ascent / descent, but the elevation map shows 5.49 km length and 12 / 17 which is kinda confusing.

I mean my other option is to merge the maps manually and upload a separate map just for that, but that kinda defeats the whole purpose of collections...

**Comment by bviktor** [comment]
To emphasize the issue even further, this is when we get into nested collections with several hundred km of maps:

![image](https://github.com/user-attachments/assets/07f77daa-57db-44f3-abf4-5cf9a6b07d96)

**Comment by morehawes** [comment]
Thanks for your suggestion, I can see what you mean about it being confusing that it initially shows the Elevation for the last Map in the Collection.

I will have to think on how this could be achieved, I have a feeling that it would be quite a bit of work though.

In the mean time you could stop the Elevation chart from being displayed initially in _Waymark Settings > Elevation > Show Initially_?

**Comment by bviktor** [comment]
Yeah, that can be a workaround, but in that case, I wouldn't even think there's even a way to show elevation, nothing indicates that, or even the fact that the map shows multiple maps at once, it's just one red line.

![image](https://github.com/user-attachments/assets/1aceaa08-01a4-4f96-b3af-d762b22f9576)

As someone who comes to the site without using Waymark before, it wouldn't be my first instinct to click randomly on the track to bring up the elevation profile :)

TBH while we're at it, this is a separate issue as well: all maps are always assigned the color red:

![image](https://github.com/user-attachments/assets/b9c4ba49-37bb-4d97-b43a-e891cb5c59cf)

Having a random color assigned to separate maps would greatly improve visual clarity imo. The way as it is now, the purpose of having this selector at all isn't clear to me.
